elm.json

```diff
@@ -0,0 +1,27 @@
+{
+    "type": "application",
```

Comment:
I don't think this should be an application. I think application `elm.json` look way different to the type for libraries.